chain/client/src/sync/sync_actor.rs

```diff
@@ -0,0 +1,43 @@
+use near_async::messaging::Sender;
+use near_primitives::hash::CryptoHash;
+use near_store::ShardUId;
+use tracing::log::info;
+
+use super::adapter::SyncMessageForClient;
+use near_network::state_sync::SyncActorMessageForNetwork;
```

Comment:
appears to be missing. Also, another nit on this and the `SyncMessageForClient`, but why not just call them something simpler like `SyncMessage`? We know it's meant for the network because it's in the `near_network` crate, and same for `SyncMessageForClient`, since it's defined by `near_client`

Reply:
I went for `SyncMessage`. My intention was to keep it clear where is this message intended to be used.

Reply:
> where ... intended to be used

A good API is about what an object does and not about for what purpose it should be used.

Reply:
> > where ... intended to be used
> 
> A good API is about what an object does and not about for what purpose it should be used.

Sure, but I think the point still stands to be honest, because what it does is already covered by the name `SyncMessage` right? Like, I think it would be strange if we renamed `near_network::types::NetworkRequests::Block` to `near_network::types::NetworkRequests::BlockForNetwork`

Reply:
@marcelo-gonzalez I agree with you and my comment supported your point :)